docs/user-guide/class-b-beaconing.md

```diff
@@ -0,0 +1,115 @@
+# Class-B: Beacon locking
+
+This doc describes the process of locking a Class-B arduino device to a beaconing signal issued from the basic station and reading the GPS coordinates transmitted by that beacon.
+
+## Network Server
+
+The LoRaWAN network server (`LoraWanNetworkSrvModule`) doesn't need any special configuration beyond the usual required launch settings.
+
+## Basic Station / Concentrator
+
+> **Note**: The following steps are not comprehensive and assume some previous knowledge of running the basic station, such as creating the `tc.uri` file, running in "NO-CUPS" mode, etc.
+
+For this setup, we use a concentrator with a GPS antenna attached.
+
+> **Note**: The GPS data can be faked by the concentrator in the case when no GPS antenna is available. **This has not yet been tested**. See [Class B Beaconing Settings](https://lora-developers.semtech.com/build/software/lora-basics/lora-basics-for-gateways/?url=conf.html), [Creating a FIFO](https://tldp.org/LDP/lpg/node17.html) and [GPS NMEA data](https://www.gpsworld.com/what-exactly-is-gps-nmea-data/#:~:text=Today%20in%20the%20world%20of,and%20match%20hardware%20and%20software.) for more information.
+
+Clone the [basic station repository](https://github.com/lorabasics/basicstation) code onto the Raspberry Pi.
+
+```bash
+git clone https://github.com/lorabasics/basicstation.git
+```
+
+Build the code:
+
+```bash
+cd basicstation
+make platform=rpi variant=std
```

Comment:
nit: I would not add instructions on how to build the basicstation executable, it might be different if user is not using a raspberry.

What about only keeping the info about station.conf?

Reply:
Okay, I made the change.